examples/lighting-app/nrfconnect/README.md

```diff
@@ -233,19 +238,22 @@ To use the native shell for setup, complete the following steps:
 
     -   Follow the
         [guide](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_assistant.html#)
-        in the nRF Connect SDK documentation to install the nRF Connect SDK
-        v1.5.0. Since command-line tools will be used for building the example,
+        in the nRF Connect SDK documentation to install the latest nRF Connect SDK
+        version. Since command-line tools will be used for building the example,
         installing SEGGER Embedded Studio is not required.
 
-    -   If you have an older version of the SDK installed, use the following
-        commands to update it to the recommended version. Replace
-        _nrfconnect-dir_ with the path to your nRF Connect SDK installation
-        directory.
+    -   If you have the SDK already installed, use the following commands
```

Comment:
I would recommend to go straight to the `--update` option to reduce number of steps.

Reply:
I agree. Please update by removing lines 248-254, so that this first line becomes the first line of the last bullet point.